pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/bestpractices/xml/ApexUnitTestClassShouldHaveAsserts.xml

```diff
@@ -115,4 +115,18 @@ public class Foo {
 }
         ]]></code>
     </test-code>
+
+    <test-code>
+      <description>#4096 [apex] api 56.0 ApexAssertionsShouldIncludeMessage and new apex class : Assert</description>
+      <expected-problems>0</expected-problems>
+      <code><![CDATA[
+@isTest
+public class Foo {
+   public static testMethod void testSomething() {
+      Account a = null;
+      Assert.areEqual(a, null, 'account not found');
+   }
```

Comment:
The test case should be improved - it tests only one of the possible assert methods. You can create multiple `testMethod` methods, each with a different assert call.
That way, we double check that e.g. we correctly identify `Assert.isNull` calls...